### PR TITLE
ENH: Import GAMIC HDF5

### PR DESCRIPTION
**Pull request by kmuehlbauer**

### CODE REVIEW

File: xradar/io/backends/odim.py [hunk 1]
```diff
@@ -768,74 +775,74 @@ def open_dataset(
...
-            "history": "None",
-            "comment": "im/exported using xradar",
-            "instrument_name": "None",
-        }
-    )
-    root = root.assign_attrs(attrs)
-    # todo: pull in only CF attributes
-    root = root.assign_attrs(sweeps[1].attrs)
-    return root
+# def _get_h5group_names(filename, engine):
```
Comment (mgrover1) [inline xradar/io/backends/odim.py:778]: Can we remove this now that this is in a different module? Instead of commenting out?

**Review by mgrover1 — COMMENTED** [review]
One minor comment - otherwise, looks great!

**Comment by kmuehlbauer** [comment]
I'll pull them in one after the other, thanks @mgrover1.

**Review by mgrover1 — APPROVED** [review]
Great! Thanks @kmuehlbauer

**Pull request merged**